src/tag_can_driver.cpp

```diff
@@ -102,6 +109,19 @@ int main(int argc, char ** argv)
 
   auto node = rclcpp::Node::make_shared("tag_can_driver");
   imu_frame_id = node->declare_parameter<std::string>("imu_frame_id", "imu");
+  auto target_frequency = node->declare_parameter<double>("target_frequency", 200.0);
+  rate_bound_status = std::make_unique<custom_diagnostic_tasks::RateBoundStatus>(
+    custom_diagnostic_tasks::RateBoundStatusParam(target_frequency * 0.95, target_frequency * 1.05),
+    custom_diagnostic_tasks::RateBoundStatusParam(target_frequency * 0.90, target_frequency * 1.10),
```

Comment:
Is it possible to expose the upper/lower range values as ROS parameters instead of fixed ones?
If you decide to expose those, as discussed in [similar modification on Nebula](https://github.com/tier4/nebula/pull/322/files#r2108157069), it would be great if you take values in absolute ones for easy tuning.